recipe/meta.yaml

```diff
@@ -1,45 +1,47 @@
 {% set name = "requirements-parser" %}
-{% set version = "0.2.0" %}
-{% set sha256 = "5963ee895c2d05ae9f58d3fc641082fb38021618979d6a152b6b1398bd7d4ed4" %}
+{% set version = "0.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/requirements_parser-{{ version }}.tar.gz
+  sha256: 588f587ab76732d59df4c64bd81f1b4a4f1aaaa9b3eb7ad4f5890685446f03e8
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
-    - python
+  host:
+    - python >=3.8,<4.0
+    - poetry-core >=1.0.0
     - pip
   run:
-    - python
-    - setuptools
+    - python >=3.8.0,<4.0.0
+    - types-setuptools >=69.1.0
```

Comment:
@wolfv should we lower this to 69.0.0.0 https://anaconda.org/conda-forge/types-setuptools?

Reply:
Just saw it got updated 🚀